website/docs/concepts/router.md

```diff
@@ -45,7 +45,7 @@ matched, the router navigates to the path with `not_found` attribute. If no rout
 a message is logged to console stating that no route was matched.
 
 Finally, you need to register the `<Router />` component as a context.
-`<Router />` provides session history information to its children.
+`<Router />` provides location information and navigator to its children.
 
 When using `yew-router` in browser environment, `<BrowserRouter />` is recommended.
```

Comment:
This bit could be expanded about  `BrowserRouter` vs `HashRouter`

Reply:
This is a deliberate choice that I made.
My rationale is that we should document the good design in the documentation.

If you really want to know everything, you should look at the API Reference (docs.rs).

It's almost never a good idea to use hash based routing and it's really a last resort when you have ran out of options.
(Poor SEO, Query string is emulated (which is also the cause of problem @Follpvosten is having), Collides with anchors, etc.)

Reply:
@futursolo Even though I agree with you, i am afraid of always having to answer this question in discord.
Perhaps we could just emphasize more that there are others. like the suggestion in other comment